frontend/resources/translations/de-DE.json

```diff
@@ -458,7 +458,7 @@
   "document.inlineLink.title.label": "Linktitel",
   "document.inlineLink.title.placeholder": "Geben Sie den Linktitel ein",
   "board.column.create_new_card": "Neu",
-  "board.menuName": "Planke",
+  "board.menuName": "Board",
```

Comment:
I haven't had Deutsch in around ten years, so it's become quite rusty, I wonder if `Brett` wouldn't be correct here?

None the less, I'm inclined to just let it be `Board`, since it's international and should make sense for most.